src/components/Projects.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Heading, Subheading } from './ComponentLibrary'
+import PageSection from './PageSection'
+import Project from './Project'
+import Wrapper from './Wrapper'
+import projectData from '../projects.json'
+
+const Projects = () => {
+  return (
+    <PageSection backgroundColor="var(--sectionbg)">
+      <Wrapper>
+        <Heading>Featured Projects</Heading>
+        <div style={{ gridTemplateColumns: '1fr 1fr', display: 'grid', gap: 30 }}>
```

Comment:
Not 100% if I am in right place, but I guess you need to adjust this grid to the mobile version and add @media